### DISCUSSION TITLE
Output images reflect input naming conventions

### DISCUSSION THREAD

**Discussion by ghost** [junyanz/pytorch-CycleGAN-and-pix2pix#1545] [Q&A] [answered]
I am using a dataset that has very specific naming conventions and would like to maintain these naming conventions when saving the "fake" imagery.  In example, exp_2002_0001.png (original named input image) and exp_2002_0001_fake_B.png (revised output reflecting new naming convention).

What would be the best way to accomplish this task?  Thanks in advance!!

**Comment by junyanz** [comment, marked as answer]
You can change this [line](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/14422fb8486a4a2bd991082c1cda50c3a41a755e/util/visualizer.py#L42). It currently reads the label (e.g., _fake_B). You can change it to your custom name. 
```python
 image_name = '%s_%s.png' % (name, label)
```

**Comment by ghost** [comment]
I will give it a try, thank you!!!
